17style-elements/styles.css

```diff
@@ -0,0 +1,135 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  margin: 0;
+  font: 100% / 1.5 sans-serif;
+  background: url("images/moon-bg.jpg") no-repeat fixed center / cover black;
+}
+.disclosure-alert {
+  background-color: hsl(0, 0%, 95%);
+  width: 85%;
+  max-width: 48rem;
+  margin: auto;
+  color: hsl(0, 0%, 20%);
+  border: 1px solid hsl(0, 0%, 0%);
+  border-radius: 1.5rem;
+  box-shadow: 0 0 0.5rem hsl(300, 40%, 5%), 0 0.5rem 6rem hsl(200, 40%, 30%),
+    0 0.5rem 10rem hsl(300, 40%, 30%);
+}
+.disclosure-header {
+  background: linear-gradient(hsl(300, 50%, 20%), hsl(300, 50%, 10%));
+  padding: 2rem 0.5rem;
+  text-align: center;
+  color: hsl(300, 50%, 95%);
+  /* different border styles add depth to this container */
+  border-top: 1px solid hsl(300, 50%, 35%);
+  border-bottom: 1px solid hsl(300, 50%, 5%);
+  border-radius: 1.375rem 1.375rem 0 0; /*a value of 1.375 rem cancels the size difference caused by the size on the left and right border of the .disclosure-alert container*/
```

Comment:
In general, I prefer code comments to go above the line they're referencing, especially if they're long like this one. There are cases where inline comments are preferred, but that's typically for things like unit conversion or shorter comments.

```scss
$sizes: (
  sm: 0.75rem, // 12px
  md: 1rem,    // 16px
  lg: 1.5rem,  // 24px
);
```

Reply:
Thank you, I appreciate you letting me know! 👍

Reply:
I also revised all the code comments accordingly. :)

Reply:
Here is the sha for the revisions I made: 8629f982ca64c1255121fc8de57c182ff6cd5fa7
I will request re-review.
Thanks again! :)